NOTICE

```diff
@@ -0,0 +1,11 @@
+cargo-pants
+Copyright 2019 Glenn Mohre, Sonatype.
+
+This product includes software developed by authors of https://github.com/sfackler/cargo-tree and 
+contains code derived from that project package distributed on GitHub. The software has been modified to:
```

Comment:
Should we mention that it is dual licensed as MIT + Apache 2.0, which makes it compatible with our own Apache 2.0 license? It might be useful to note because it wouldn't show up in an automated output the same way that it would if we includeded `cargo-tree` as a dependency in the `Cargo.toml`.

Reply:
I can do that!

Reply:
In: 7f380f6